rpcserver.go

```diff
@@ -7321,10 +7321,28 @@ func (r *rpcServer) DebugLevel(ctx context.Context,
 		return nil, err
 	}
 
+	subLoggers := r.cfg.LogWriter.SubLoggers()
```

Comment:
nit: since we are here we might as well move this cli under the group `Debug: `?

Reply:
hmm are you sure, currently we only have one debug option flag ?